fab/utils/prioritised_replay_buffer.py

```diff
@@ -43,14 +45,15 @@ def __init__(self, dim: int,
         self.current_index = 0
         self.is_full = False  # whether the buffer is full
         self.can_sample = False  # whether the buffer is full enough to begin sampling
+        self.sample_with_replacement = sample_with_replacement
```

Comment:
Set to false by default (as we don't want to have repeated samples in the batch passed to the flow update)

Reply:
I have added the option now if we want to play around, but is default set to what we had before